teleport/assets/logs/teleport_tests.yaml

```diff
@@ -0,0 +1,216 @@
+id: teleport
+tests:
+  - sample: >
+      2024-05-28T14:21:41Z WARN [PROXY:1:C] Cache was forced to load session
+      web_session/936cdcda1df1a7a97e5ce2239bd39f71d7e7524a29560fdfdbf206e342f8fddc
+      from upstream. Frequent occurrence may indicate sync/perf issues.
+      cache/cache.go:2494
+    result:
+      custom:
+        date: 1716906101000
```

Comment:
Can you please add a `date-remapper` to the pipeline so this is used as the official date attribute? You can see an example [here](https://github.com/DataDog/integrations-core/blob/fd3251ed14a86bbbbff1e9678a2f5321db12e58f/ceph/assets/logs/ceph.yaml#L56-L60).

We cannot rely on date attributes [here](https://docs.datadoghq.com/logs/log_configuration/pipelines/?tab=date#date-attribute) because the log is not JSON.

Without this change the official timestamp used for querying and displaying the log will actually be the time Datadog received the log and not the time parsed from the log.

Reply:
Done!
I thought that since `date` field was part of the [standard date attributes](https://docs.datadoghq.com/logs/log_configuration/pipelines/?tab=date#date-attribute), it would be automatically picked up, but you are right, that only works for JSON logs/attributes.